pkg/skaffold/errors/errors.go

```diff
@@ -27,28 +27,17 @@ import (
 
 // These are phases in a DevLoop
 const (
-	Init        = Phase("Init")
 	Build       = Phase("Build")
 	Deploy      = Phase("Deploy")
 	StatusCheck = Phase("StatusCheck")
 	FileSync    = Phase("FileSync")
 	DevInit     = Phase("DevInit")
 	Cleanup     = Phase("Cleanup")
-
```

Comment:
you are test failures are due to this unintentional revert

Reply:
Reverted the change.